packages/strapi-helper-plugin/lib/src/components/LeftMenuLink/index.js

```diff
@@ -4,11 +4,11 @@ import { NavLink } from 'react-router-dom';
 
 import Icon from './Icon';
 
-function LeftMenuLink({ children, to }) {
+function LeftMenuLink({ children, to, Component }) {
   return (
     <NavLink to={to}>
       <Icon />
-      <p>{children}</p>
+      {Component ? <Component /> : <p>{children}</p>}
```

Comment:
We should replace the `<p>{children}</p>` by `{children}` to avoid the `Component` props.

Reply:
If this update needs a lot of modifications in the codebase, can you just put a comment about that, please 😉

Reply:
This menu is also used in the settings... In the first place I didn't want to add any modification to this component which clearly needs some improvements but I got stuck!

Let's write one more TODO comment...